src/render.tsx

```diff
@@ -6,13 +6,19 @@ import { createHtmlStylesheet, HtmlStyle, HtmlStyles } from './styles';
 import { Style } from '@react-pdf/types';
 import { isText, Tag } from './tags';
 
-export type HtmlRenderer = React.FC<
-  React.PropsWithChildren<{
-    element: HtmlElement;
-    style: Style[];
-    stylesheets: HtmlStyles[];
-  }>
->;
+export type BaseHtmlRenderer = {
```

Comment:
I think you could just name this HtmlRendererProps and remove children from it.

Reply:
Renamed, however renderSvgs relies on the children prop so i left it the way it is.